evap/staff/views.py

```diff
@@ -2141,6 +2147,23 @@ def user_delete(request):
     return HttpResponse()  # 200 OK
 
 
+@require_POST
+@manager_required
+def user_resend_email(request):
+    user = get_object_from_dict_pk_entry_or_logged_40x(UserProfile, request.POST, "user_id")
+
+    template = EmailTemplate.objects.get(name=EmailTemplate.EVALUATION_STARTED)
+    body_params = {
+        "user": user,
+        "evaluations": {},
+        "due_evaluations": user.get_sorted_due_evaluations(),
```

Comment:
Please fill `evaluations` and leave `due_evaluations` empty as described in the issue. Currently, the generated email doesn't really make sense :)